samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/servlet/evaluation/ExportReportServlet.java

```diff
@@ -115,18 +116,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             return;
         }
 
-        String reportTitle = EVENT_LOG_BUNDLE.getFormattedMessage("item_analysis") + ": " + highestSubmissionHsBean.getAssessmentName();
+        String reportTitle = EVALUATION_BUNDLE.getFormattedMessage("item_analysis") + ": " + highestSubmissionHsBean.getAssessmentName();
+        Optional<String> optSiteTitle = getSite(assessment.getOwnerSiteId()).map(Site::getTitle);
+        String reportSubject = optSiteTitle.map(siteTitle -> EVALUATION_BUNDLE.getFormattedMessage("export_site") + ": " + siteTitle).orElse(null);
```

Comment:
Maybe safer to use an empty Strin in the orElse?

Reply:
If I pass null the subject line not is displayed. If I pass an empty string it would show an extra empty line.